src/components/map/overview-table.tsx

```diff
@@ -265,7 +267,10 @@ export function OverviewTable({
     React.useState<VisibilityState>({});
   const [rowSelection, setRowSelection] = React.useState({});
 
-  const tableData = aggregatePlayerData(playerStats);
+  const tableData = React.useMemo(
+    () => aggregatePlayerData(playerStats),
+    [playerStats]
+  );
```

Comment:
This is the main fix. The results of `aggregatePlayerData` needed to be memoized, otherwise I think it was getting caught in an infinite render loop. But I never did receive a "Maximum depth exceeded" error, and I'm not sure why I didn't. Not sure if it's a turbopack or app router thing.

Reply:
Turbopack is disabled in dev as of a few days ago because of issues with importing fonts - could definitely be an app router thing or just something I missed.